docs/APIs/io/BusInOut.md

```diff
@@ -1,8 +1,8 @@
 #BusInOut
 
-The BusInOut interface is used as a bi-directional bus that collects together a number of [DigitalInOut](DigitalInOut.md) pins that can be read and written as one value.
+Use the BusInOut interface as a bidirectional bus that collects a number of [DigitalInOut](DigitalInOut.md) pins that can be read and written as one value.
```

Comment:
Well, I suppose if we always say we read and write a pin, not read from and write to, then you can ignore my comment on the previous PR